src/ensembl/src/global/globalHelper.ts

```diff
@@ -3,12 +3,16 @@ import { BreakpointWidth } from './globalConfig';
 export const getBreakpoint = (
   key: keyof typeof BreakpointWidth
 ): BreakpointWidth => {
-  if (BreakpointWidth[key] >= BreakpointWidth.DESKTOP) {
+  if (BreakpointWidth[key] >= BreakpointWidth.BIG_DESKTOP) {
+    return BreakpointWidth.BIG_DESKTOP;
+  } else if (BreakpointWidth[key] >= BreakpointWidth.DESKTOP) {
     return BreakpointWidth.DESKTOP;
   } else if (BreakpointWidth[key] >= BreakpointWidth.LAPTOP) {
     return BreakpointWidth.LAPTOP;
-  } else {
+  } else if (BreakpointWidth[key] >= BreakpointWidth.TABLET) {
     return BreakpointWidth.TABLET;
+  } else {
+    return BreakpointWidth.PHONE;
   }
 };
```

Comment:
It looks like could be as simple as saying `return BreakpointWidth[key]`. 
@ima23 if this is too much to change in this PR, could you create a ticket to clean this up, please?

Reply:
@msmanoj I'll create a ticket for this as per our discussion offline